### PR TITLE
Feat/dark theme

### PR DESCRIPTION
**Pull request by bighnesh0404**
### Functionality:
Implemented dark theme to the website

### Solution:
Dark theme has been implemented to the worktrolly website

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
ng serve 
s1
s2
s3
s4
s5

### CODE REVIEW

File: worktrolly-ui/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "worktrolly-ui",
-  "version": "0.0.0",
+  "name": "worktrolly",
+  "version": "0.2.0",
```
Comment (shivamgoyal7) [inline worktrolly-ui/package-lock.json:3]: undo this

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
Also, change the "Board" text color to white in the board component.

**Comment by bighnesh0404** [comment]
> Also, change the "Board" text color to white in the board component.
Bhaiya told to leave that because it will be handled by some other member

File: worktrolly-ui/package-lock.json (hunk 1 above)
Comment (bighnesh0404) [reply worktrolly-ui/package-lock.json:3]: Okay sure

**Review by shivamgoyal7 — APPROVED** [review]
ship it

**Review by viv9k — APPROVED** [review]
ship it

**Comment by Shrishti1802** [comment]
Resolve merge conflicts

**Pull request merged**